### DISCUSSION TITLE
알수없음 답장 뿌튀 관련

### DISCUSSION THREAD

**Discussion by cafemene** [storycraft/node-kakao#811] [Q&A] [answered]
안녕하세요. 최근 "(알 수 없음)에게 답장" 형식의 광고로 고통받고 있는 사람입니다..

![image](https://user-images.githubusercontent.com/52375776/142762360-047b584e-5407-4c43-a2e3-b664df62cad1.png)

위와같은 형식으로 오픈 챗방에서 광고를 무자비하게 뿌리고 있어서
메시지를 분석하고 있는데 data에서 어떤 파라미터에도 걸리지 않는 상황입니다 ㅠㅠ

혹시 어디쪽을 참고하면 될지 힌트라도 얻을수 있을까요?

답변 부탁드립니다 ㅠㅠ

**Comment by ghost** [comment, marked as answer]
https://github.com/storycraft/node-kakao/issues/763

**Reply by cafemene** [reply]
감사합니다!

**Reply by je0404** [reply]
답장 뿌튀때문에 저도 고통 받고 있는데 
getChatListFrom메서드만이 현재로서는 유일한 방법인가요??
얘네들이 한번에 폭탄도배를 해버리니까 getChatListFrom에서도 메세지가 남지 않는 경우가 있어서요
userid도 안잡히고..
